tests/test_v1.py

```diff
@@ -408,3 +408,39 @@ def test_cancel_job_good(mongo_app):
     assert "OK" == output.text
     job = mongo.jobs.find_one({"job_id": job_id})
     assert job["result_data"]["job_state"] == "cancelled"
+
+
+def test_agents_post(mongo_app):
```

Comment:
I guess it would be good to have “failing test(s)” as well

Reply:
done.

Reply:
Do I miss it? Cannot find the changes. I mean, we test the case when the data is valid and we get 200 code. Shouldn't we also have a test when the data is not valid?

Reply:
Thanks!